src/character.h

```diff
@@ -241,8 +243,8 @@ class Character : public Creature, public visitable<Character>
         /** Toggles a trait on the player and in their mutation list */
         void toggle_trait(const std::string &flag);
         /** Add or removes a mutation on the player, but does not trigger mutation loss/gain effects. */
-        void set_mutation(const std::string &flag);
```

Comment:
Now that I think about it, do we need the warning version at all?
It only warns about mutation being set/unset more than once, which isn't really an error worth caring about. In fact, it would be more convenient to allow setting a set mutation to avoid `if(!has_trait)set`.

I think I'll just drop the warnings.